document/api/src/main/java/io/camunda/document/api/DocumentMetadataModel.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.document.api;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+public record DocumentMetadataModel(
+    String contentType,
+    String fileName,
+    ZonedDateTime expiresAt,
```

Comment:
Any reason not to use a timestamp (long) here?

Reply:
As per the [API guidelines](https://docs.google.com/document/d/1G9AmmNac-4QLGZ0LXQXa3FyeCrSJdIboaPt3-R6dWNw/edit#heading=h.yilyqujdhsix) we cannot return a timestamp from the API - consequently, returning it as a Date object here saves us an extra conversion. I wasn't sure about `ZonedDateTime` vs `LocalDateTime` but all other endpoints seem to use `ZonedDateTime`.

Reply:
Somehow it got mixed up. I meant the filesize. 😄

Reply:
Replied re. filesize [here](https://github.com/camunda/camunda/pull/21965#discussion_r1751590270), long story short I think it will always be present in the payload so no need to make it nullable